seed_data/seedData.js

```diff
@@ -50,29 +50,22 @@ for (var i = freelancers.data.length - 1; i >= 0; i--) {
   var rndl = Math.floor(Math.random() * locations.length);
   freelancers.data[i].address = locations[rndl];
 }
-// hardcode location for availability
-for (var i = availabilities.data.length - 1; i >= 0; i--) {
-  var rndl = Math.floor(Math.random() * locations.length);
-  availabilities.data[i].location = locations[rndl];
-}
+
 // The first 'staticUsers' users are static and can be used for tests
 const staticUsers = 6;
 // bind non-static users to Freelance profiles
-for (var i = staticUsers; i < usersSize; i++) {
-  var rndf = Math.floor(Math.random() * freelancersSize);
+for (var i = staticUsers, k = 0; i < usersSize; i++) {
+  // var rndf = Math.floor(Math.random() * freelancersSize);
   users.data[i].freelancer = freelancers.data[i]._id;
   freelancers.data[i].owner = users.data[i]._id;
   // give non-static users an availability schedule
-  for (var j = 0; j < 10; j++) {
-      // var rndu = staticUsers + Math.floor(Math.random() * (usersSize - staticUsers);
-      var d = new Date(new Date(availabilities.data[j].begin).getUTCFullYear(),new Date(availabilities.data[j].begin).getUTCMonth(),new Date(availabilities.data[j].begin).getUTCDay(),12,0,0,0);
-      var a = {
-        day: d,
-        begin: new Date(availabilities.data[j].begin),
-        end: new Date(availabilities.data[j].end),
-        location: availabilities.data[j].location
-      }
-      freelancers.data[i].availability.push(a);
+  for (var j = 0; j < 10; j++, k++) {
```

Comment:
`j` restarts from 0 several times, which results in a repetition of availability data for each freelancer. Using `k`, each freelancer gets different availabilities.